packages/cloudflare/src/mappers/request.mapper.ts

```diff
@@ -1,21 +1,21 @@
 import {injectable} from "tsyringe";
```

Comment:
Should we just delete this class ?

Reply:
No, it's in progress, I'll complete it later on.